xml/core_registers.xml

```diff
@@ -22,7 +32,15 @@
             When 1, {\tt ebreak} instructions in User/Application Mode enter
             Debug Mode.
         </field>
-        <field name="0" bits="11" access="R" reset="0" />
+        <field name="intdisable" bits="11" access="R/W" reset="0">
+            0: Interrupts are handled as usual.
+
+            1: Interrupts are disabled. This bit supersedes mstatus.ie.
+
+            This feature may not exist in implementations from early in the
```

Comment:
The statement that interrupts are masked during Debug Mode:
https://github.com/riscv/riscv-debug-spec/blob/0512f5d9e9f80b392b91ca32c17f493544820302/core_debug.tex#L17

I think this is a statement we don't want to change.

Then there are 2 separate functionalities we are discussing:

1) The statement here says that interrupts are disabled while single-stepping. I envisioned that this bit would only be an "IE during single stepping":
https://github.com/riscv/riscv-debug-spec/blob/1fbbe6e955e4cbfd478c1a07d33e2106bf65670e/xml/core_registers.xml#L69

2) An entirely different functionality, which creates another bit "DIE" which when 0 would disable interrupts, regardless of whether in debug mode or not. That is a completely separate issue from 1 with a larger scope.